source/languageHandler.py

```diff
@@ -23,6 +24,12 @@
 LOCALE_SLANGDISPLAYNAME=0x6f
 LOCALE_USER_DEFAULT = 0x400
 LOCALE_CUSTOM_UNSPECIFIED = 0x1000
+LOCALE_SENGLISHLANGUAGENAME = 0x00001001
+LOCALE_SENGLISHCOUNTRYNAME = 0x00001002
+LOCALE_IDEFAULTANSICODEPAGE = 0x00001004
+
+CP_ACP = "0"

```

Comment:
This variable name isn't very descriptive, is this an accurate description?

```suggestion
CODEPAGE_ANSICODEPAGE = "0"
```

Reply:
This is how Microsoft names this constant and for someone familiar with the encoding handling in Windows this is pretty clear. I've hopefully clarified with a comment (853f765a2) let me know if it makes things better.